src/mono/wasm/Makefile

```diff
@@ -43,6 +43,10 @@ EMCC=source $(EMSDK_PATH)/emsdk_env.sh 2>/dev/null && emcc
 provision-wasm: .stamp-wasm-install-and-select-$(EMSCRIPTEN_VERSION)
 	@echo "----------------------------------------------------------"
 	@echo "Installed emsdk into EMSDK_PATH=$(TOP)/src/mono/wasm/emsdk"
+	@if [ $(wildcard /.dockerenv) != "" ]; then\
+        echo 'export NO_SANDBOX_IN_CONTAINER="--no-sandbox"' >> ~/.bashrc;\
```

Comment:
This feels wrong - having the makefile writing to the `~/.bashrc` file. Can the test code itself check if we are running in a docker container, and then add the switch?